go/pkg/cosmos/handler.go

```diff
@@ -28,6 +28,7 @@ type RouteHandler interface {
 	GetInfo() (api.Info, error)
 	GetAccount(pubkey string) (api.Account, error)
 	GetTxHistory(pubkey string, cursor string, pageSize int) (api.TxHistory, error)
+	GetValidatorTxHistory(pubkey string, cursor string, pageSize int) (api.TxHistory, error)
```

Comment:
In the same vein, remove this function from the interface as it should align with the functions declared in `pkg/cosmos/api.go`. It would probably make more sense for this interface to be declared in there as well so it can be made more clear what the interface is directly associated with.
```suggestion
```